### PR TITLE
Bboxlegendfix

### PR DESCRIPTION
**Pull request by voisardf**
Fixed the extension of the "other legend" overlay to the map window bbox

### CODE REVIEW

File: crdppf/lib/content.py [hunk 1]
```diff
@@ -7,7 +7,7 @@
 from crdppf.lib.wmts_parsing import wmts_layer
 from crdppf.lib.geometry_functions import get_feature_bbox, get_print_format, get_feature_center
 
-from crdppf.views.get_features import get_features_function
+from crdppf.util.get_feature_functions import get_features_function
```
Comment (kalbermattenm) [inline crdppf/lib/content.py]: It a bit weird to have <pre>get_feature_function<b>s</b></pre> as file name and <pre>get_feature<b>s</b>_function</pre> as function name. Isn't it ?

File: crdppf/lib/content.py [hunk 2]
```diff
@@ -61,7 +92,7 @@ def set_documents(request, topicid, doctype, docids, featureinfo, geofilter, top
                 documents.append({"officialtitle": doc['officialtitle'], "title": doc['title'], "remoteurl": doc['remoteurl']})
                 if doc['doctype'] != u'legalbase' and doc['documentid'] not in doclist:
                     appendices.append(add_appendix(topicid, 'A'+str(len(appendices)+1), unicode(doc['officialtitle']).encode('iso-8859-1'),
-                        unicode(doc['title']).encode('iso-8859-1'), unicode(
+                                                   unicode(doc['title']).encode('iso-8859-1'), unicode(
```
Comment (kalbermattenm) [inline crdppf/lib/content.py]: This is a huge indent...

Comment (voisardf) [reply crdppf/lib/content.py]: I agree, but flake says, that is, what's needed... 😀 ... so who am I to object... 😉

File: crdppf/lib/content.py (hunk 1 above)
Comment (voisardf) [reply crdppf/lib/content.py]: That's historical... I'll might change it if we keep v1 longer

**Comment by voisardf** [comment]
Thanks a lot, merging!

**Pull request merged**